app/workers/sha_poller_worker.rb

```diff
@@ -0,0 +1,17 @@
+# Polls weather for subscribed locations
+class ShaPollerWorker
+  include Sidekiq::Worker
+  include Sidekiq::Repeat::Repeatable
+
+  repeat { hourly.minute_of_hour(0, 5, 10, 15, 20, 25, 30, 35, 40, 45, 50, 55) }
+
+  def perform
+    logger.info "Running sha poll"
+
+    Helios2Schema.subscriptions.trigger(
+      "schemaShaPublished",
+      {},
+      AppEnv['HEROKU_SLUG_COMMIT']
```

Comment:
Let's make sure to update the `env.sample` file since we're depending on a new env variable now.

Reply:
I'm on the fence with this. `HEROKU_SLUG_COMMIT` should only exist in the heroku instance. I guess we'd make a blank example in case a developer wanted to test? I don't think its worthwhile for us to integrate git into rails to look up just for local sha updates.